pom.xml

```diff
@@ -96,18 +96,18 @@
 			<dependency>
 				<groupId>javax.servlet</groupId>
 				<artifactId>javax.servlet-api</artifactId>
-				<version>3.1.0</version>
+				<version>4.0.1</version>
 				<scope>provided</scope>
 			</dependency>
 			<dependency>
 				<groupId>javax.servlet</groupId>
 				<artifactId>jsp-api</artifactId>
-				<version>2.0</version>
+				<version>2.0.public_draft</version>
```

Comment:
Was this among the ticket description, adding 2.0.public _draft

Reply:
yes @sherrif10

Reply:
Please get rid of 2.0.publicc_draft

Reply:
ok i will remove   2.0.public_draft version

Reply:
Yes, public_draft is the older version. Leave it as 2.0